docker/opbeans/rum/tasks.js

```diff
@@ -37,4 +37,11 @@ async function run() {
     }
 }
 
-run().catch(console.error.bind(console))
+try {
+  run();
+} catch (e) {
+  if (e.message.includes("timed out after")) {
+    return run();
+  }
+  console.error.bind(console);
+}
```

Comment:
What you had before was better. This will only catch the first timeout. Additional timeouts will still make the process crash.

Reply:
There was a discussion earlier to avoid using the recursion as that particular implementation could potentially end as an infinite loop, refers to:
- https://github.com/elastic/apm-integration-testing/pull/664#discussion_r345300191
- https://github.com/elastic/apm-integration-testing/pull/664#discussion_r345293432

No strong opinion about what's the best strategy though, probably I misunderstood what you meant in your comment: https://github.com/elastic/apm-integration-testing/pull/664#discussion_r346056629

@kuisathaverat  and @mdelapenya what're your thoughts if we keep the first approach with the suggested improvement : https://github.com/elastic/apm-integration-testing/pull/664#discussion_r346056629 ?

Reply:
I'll buy the Soren solution https://github.com/elastic/apm-integration-testing/pull/664#issuecomment-553860528 😃

Reply:
BTW https://twitter.com/johnwilander/status/1176457013305303040?lang=en